automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -28,169 +30,468 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
 
 
   "A workspace with an authorization domain" - {
+    "with one group inside of it" - {
+      "can be created" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec") { authDomainName =>
```

Comment:
yes this was intended, it also signs in as that same user when creating the workspace